src/main/java/uk/gov/hmcts/divorce/common/event/page/ConditionalOrderReviewAoSApplicant2.java

```diff
@@ -25,9 +25,13 @@ public void addTo(PageBuilder pageBuilder) {
 
         pageBuilder
             .page("ConditionalOrderReviewAoSApplicant2", this::midEvent)
-            .pageLabel("Review Acknowledgement of Service - Draft Conditional Order Application")
```

Comment:
Don’t we need page label ?

Reply:
Michael said he wanted them removed - I did it for the Applicant 1 page on the Withdraw Joint Application ticket last week.

Reply:
Just doing it for the Applicant 2 one here